styles/vendor/magento-ui/_forms.scss

```diff
@@ -36,37 +36,70 @@ $form-element-input-placeholder__font-style: $form-element-input__font-style !de
 $form-validation-note__color-error         : $error__color !default;
 $form-element-validation__border-error     : lighten($form-validation-note__color-error, 20%) !default;
 
+$form-element-input-disabled__background   : $form-element-input__background !default;
+$form-element-input-disabled__border       : $form-element-input__border !default;
+$form-element-input-disabled__opacity      : 0.5 !default;
+$form-element-input-disabled__color        : $form-element-input__color !default;
+$form-element-input-disabled__font-style   : $form-element-input__font-style !default;
+
+$form-element-input-focus__background      : $form-element-input__background !default;
+$form-element-input-focus__border          : $form-element-input__border !default;
+$form-element-input-focus__color           : $form-element-input__color !default;
+$form-element-input-focus__font-style      : $form-element-input__font-style !default;
+
+$note__color                               : $error__color !default;
+$note__font-size                           : $font-size__s !default;
+$note__font-family                         : inherit !default;
+$note__font-style                          : inherit !default;
+$note__font-weight                         : inherit !default;
+$note__line-height                         : inherit !default;
+$note__margin                              : 3px 0 0 !default;
+$note__padding                             : inherit !default;
+$note__icon-use                            : inherit !default;
+
+$note-icon__font-content                   : $icon-pointer-up !default;
+$note-icon__font                           : $icon-font !default;
+$note-icon__font-size                      : $font-size__s * 2 !default;
+$note-icon__font-line-height               : $font-size__s * 2 !default;
+$note-icon__font-color                     : $error__color !default;
+$note-icon__font-color-hover               : inherit !default;
+$note-icon__font-color-active              : inherit !default;
+$note-icon__font-margin                    : inherit !default;
+$note-icon__font-vertical-align            : $icon-font__vertical-align !default;
+$note-icon__font-position                  : $icon-font__position !default;
+$note-icon__font-text-hide                 : $icon-font__text-hide !default;
+
 @mixin lib-form-element-input($_type) {
-    $_background                : $color-white;
-    $_border                    : 1px solid $form-element-input__border-color;
-    $_border-radius             : 1px;
+    $_background                : $form-element-input__background;
```

Comment:
We can't take `$form-element-input` as default for every type of element.
Let's check how the LESS code looks like - https://github.com/magento/magento2/blob/develop/lib/web/css/source/lib/_forms.less

They are using variable `$_type` to create new variable name for every type. 
In SASS using same structure is prohibited, so if we had to create a set of variables for every type of input manualy.